tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-        py{27,35}-django{111}-drf{37,38,39,latest},py{35}-django{20,21,22}-drf{37,38,39,latest},
+        py{35}-django{20,21,22}-drf{37,38,39,latest},
```

Comment:
The set of DRF versions being tested should be updated now that edx-platform is on 3.9.4.  Maybe drop 3.7 and 3.8 and add 3.10 and 3.11.  The versions below would need to be updated accordingly.